db/models.py

```diff
@@ -1 +1,35 @@
 from django.db import models
+
+
+class Race(models.Model):
+    RACES = (
+        ("Elf", "elf"),
+        ("Human", "human"),
+        ("DwarfF", "dwarf"),
```

Comment:
There seems to be a typo in the race name 'DwarfF'. It should probably be 'Dwarf'.